### PR TITLE
Fixes #10656 - Prevent audits on counter cache update

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by dLobatog** [comment]
@tbrisker Seems to break on Katello, otherwise :+1:

**Comment by domcleal** [comment]
[test] that was the issue Eric mentioned on the -dev list.

**Comment by domcleal** [comment]
Is this an issue in Architecture, LookupKey, Puppetclass or Realm models?  They all have counters and are audited without exclusions.

I tried to reproduce the bug with host to arch but couldn't, is there another reason it might not happen on those?

**Comment by tbrisker** [comment]
@domcleal I couldn't reproduce for those either, but I'm adding the exclusion anyways to be on the safe side (except Puppetclass which was already fixed in #2395).

File: app/models/lookup_key.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ class LookupKey < ActiveRecord::Base
   KEY_DELM = ","
   EQ_DELM  = "="
 
-  audited :associated_with => :audit_class, :allow_mass_assignment => true
+  audited :associated_with => :audit_class, :allow_mass_assignment => true, :except => [:hosts_count, :hostgroups_count]
```
Comment (domcleal) [inline app/models/lookup_key.rb]: lookup_values_count?

Comment (tbrisker) [inline app/models/lookup_key.rb]: Note to self: think before you copy&paste.

**Comment by domcleal** [comment]
Merged as 09f3131, thanks @tbrisker.

**Pull request closed without merging**